trie/triedb/pathdb/disklayer.go

```diff
@@ -255,25 +331,26 @@ func (dl *diskLayer) revert(h *history, loader triestate.TrieLoader) (*diskLayer
 }
 
 // setBufferSize sets the node buffer size to the provided value.
-func (dl *diskLayer) setBufferSize(size int) error {
-	dl.lock.RLock()
-	defer dl.lock.RUnlock()
-
-	if dl.stale {
-		return errSnapshotStale
-	}
-	return dl.buffer.setSize(size, dl.db.diskdb, dl.cleans, dl.id)
-}
+//func (dl *diskLayer) setBufferSize(size int) error {
+//	dl.lock.RLock()
+//	defer dl.lock.RUnlock()
+//
+//	if dl.stale {
+//		return errSnapshotStale
+//	}
+//	return dl.buffer.setSize(size, dl.db.diskdb, dl.cleans, dl.id)
+//}
 
 // size returns the approximate size of cached nodes in the disk layer.
```

Comment:
The annotation doesn't quite match the function return. Please correct it.

Reply:
handled